src/Util/AdminObjectAclManipulator.php

```diff
@@ -285,12 +284,11 @@ protected function buildForm(AdminObjectAclData $data, FormBuilderInterface $for
                 ];
             }
 
-            // NEXT_MAJOR: remove when dropping Symfony <2.8 support
-            $type = method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix') ?
-                'Sonata\AdminBundle\Form\Type\AclMatrixType' :
-                new AclMatrixType();
-
-            $formBuilder->add($key, $type, ['permissions' => $permissions, 'acl_value' => $aclValue]);
+            $formBuilder->add(
+                $key,
+                'Sonata\AdminBundle\Form\Type\AclMatrixType',
```

Comment:
Please use `AclMatricType:class`

Please use `::class` constant wherever needed in your PR.
Please do it in a separate commit.

Reply:
Sorry, I didn't saw the comment, that  you will fix the class names later.